src/dos/cdrom_ioctl_linux.cpp

```diff
@@ -357,4 +363,9 @@ std::vector<int16_t> CDROM_Interface_Ioctl::ReadAudio(const uint32_t sector, con
 	return audio_samples;
 }
 
+bool CDROM_Interface_Ioctl::HasDataTrack() const
+{
+	return true; /*TODO*/
```

Comment:
This is pretty simple to implement (very similar to what you did on Win32).  I can do it once this merges if you don't have a Linux machine.

Reply:
No Linux-machine at the moment. I'll see if I can whip up a VM somewhere in the near future, but I would like to invite you to implement this one for now. 😄

Reply:
Let's merge this first, then @weirddan455 can tinker with it (if he wants).